### ISSUE TITLE
Improve error returns by the API

### ISSUE THREAD

**Issue by zaibon** [zero-os/0-orchestrator#1033]
Sometimes, the API returns an error that says: `error waiting on run XXXXXXXXX`
This message doesn't give any information about the error, It would be nice to have more detail. 
AYS let you inspect the logs of every jobs, so it should be possible to the error that happens within a run
